frontend/packages/ux-editor/src/components/config/Expressions/SimpleExpression.tsx

```diff
@@ -27,11 +27,11 @@ export const SimpleExpression = ({
   const t = useText();
   return (
     <>
-      {expression.subExpressions.map((subExp: SubExpression, index: number) => (
+      {expression.subExpressions?.map((subExp: SubExpression, index: number) => (
         <div key={subExp.id}>
```

Comment:
Her kan du ta i bruk `React.Fragment` i stedet for `div`, siden det ikke gir mening å gruppere disse elementene i DOM-en. (Det er det samme som `<></>`, bare at man også kan ha attributter som `key` på den.)